<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu_1</class>
 <widget class="QMainWindow" name="MainMenu_1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>443</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>337</width>
    <height>443</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>337</width>
    <height>443</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string notr="true">23:59 main menu</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(90, 90, 90);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>291</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="exit_btn">
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(61, 61, 61);
color: rgb(216, 216, 216);</string>
       </property>
       <property name="text">
        <string>exit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="start_btn">
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(61, 61, 61);
color: rgb(216, 216, 216);</string>
       </property>
       <property name="text">
        <string>start</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>195</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="sounds_btn">
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(61, 61, 61);
color: rgb(216, 216, 216);</string>
       </property>
       <property name="text">
        <string>sounds</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="music_btn">
       <property name="cursor">
        <cursorShape>CrossCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(61, 61, 61);
color: rgb(216, 216, 216);</string>
       </property>
       <property name="text">
        <string>music</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>160</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="music_pos1">
       <property name="styleSheet">
        <string notr="true">
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>music 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="music_pos2">
       <property name="styleSheet">
        <string notr="true">color: rgb(216, 255, 255);</string>
       </property>
       <property name="text">
        <string>music 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="music_pos3">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 216, 255);</string>
       </property>
       <property name="text">
        <string>music 3</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="music_pos4">
       <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 216);</string>
       </property>
       <property name="text">
        <string>music 4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>337</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
